chrome/content/zotero/xpcom/sync/syncLocal.js

```diff
@@ -144,9 +144,9 @@ Zotero.Sync.Data.Local = {
 			if (io.accept) {
 				var resetDataDirFile = OS.Path.join(Zotero.DataDirectory.dir, 'reset-data-directory');
 				yield Zotero.File.putContentsAsync(resetDataDirFile, '');
-				
+
+				Zotero.Prefs.resetBranch([], 'extensions.zotero.sync.storage.groups.');
```

Comment:
In addition to this, we should clear the groupID-specific one when a group is erased — probably in `_finalizeErase()` in group.js.